api/base/filters.py

```diff
@@ -218,7 +218,22 @@ def parse_query_params(self, query_params):
                                 'source_field_name': source_field_name
                             }
                         })
-
+                    elif source_field_name == 'is_preprint':
+                        op = 'eq' if value == 'false' else 'ne'
+                        query.get(key).update({
+                            field_name: {
+                                'op': op,
+                                'value': None,
+                                'source_field_name': 'preprint_file'
+                            }
+                        })
+                        query['_is_preprint_orphan'] = {
```

Comment:
I'm not certain this part works -- `_is_preprint_orphan` is set when a node is loaded into memory and the `is_preprint` property is checked, but I think it's currently not persisting because it's not being saved.